src/tests/execute/create_ask_tests.rs

```diff
@@ -922,6 +909,20 @@ mod create_ask_tests {
             },
         );
 
+        QueryAttributesRequest::mock_response(
```

Comment:
is this supposed to be here?

Reply:
Yes because with the version upgrade, there isn't a default mock so the `get_attributes()` will just fail when the test is ran. 

The expected behavior of this test is that when `get_attributes()` is called, it returns an empty list. That then causes the expected error to happen [here](https://github.com/FigureTechnologies/ats-smart-contract/blob/4815290f0d06d7a7a8c9532c21a5eddc43ecfa16/src/contract.rs#L424) when the required attributes are checked. This is the behavior on the main branch. 

I've put in the mock here to return one of the required attributes but not both (I found it to be a little clearer of a test), so it will still error on the expected line. I did check that the same behavior is true in the main branch too. Let me know if you want it to return an empty list instead.